### ISSUE TITLE
features.autoFocus not working

### ISSUE THREAD

**Issue by nonsenseusername** [okta/okta-signin-widget#1850]
`feature.autoFocus` won't work, since its not referenced in code but mentioned in documention.

### I'm submitting a
  - [x] bug report
 

### Background info
User is unable to disable autoFocus after widget is rendered, the reason being
the code has been removed, please refer to this pull request.
[pull request](https://github.com/okta/okta-signin-widget/pull/720/files#diff-d227892855474f716aac731e2d1711185f560018288a52c44e9af3661528111bl)

**Comment by denysoblohin-okta** [comment]
Thanks for submitting this issue.
[The PR you mentioned](https://github.com/okta/okta-signin-widget/pull/720) was not merged, and `autoFocus` is not documented in readme.
Do you want to request a feature to disable autofocus?

**Comment by nonsenseusername** [comment]
yes, in mobile devices autofocus causes the keyboard to popup, which is not desired

**Comment by denysoblohin-okta** [comment]
Internal ref: [OKTA-389004](https://oktainc.atlassian.net/browse/OKTA-389004)

**Comment by kushal98** [comment]
Hello,
Is this issue resolved and the autofocus feature available now ?

**Comment by nonsenseusername** [comment]
in the meantime, i use the script before build process to remove autofocus
```
try {
  const RGX = /(focus:function\(\){)(this\.model\.get\("username"\))/;
  const SRCFILE = "node_modules/@okta/okta-signin-widget/dist/js/okta-sign-in.entry.js";
  const filee = await fs.readFile(SRCFILE, "utf8");
  await fs.writeFile(SRCFILE, filee.replace(RGX, "$1 return; $2"), "utf-8");
} catch (ex) {
  console.error("Error ", ex);
}
```

**Comment by deepamahalingam-okta** [comment]
@nonsenseusername thanks for posting this workaround. Did the workaround resolve your issue?

**Comment by deppensteiner** [comment]
Any updates here?

**Comment by denysoblohin-okta** [comment]
Unless you use [ClassicEngine](https://github.com/okta/okta-signin-widget#useclassicengine) (v1), `features.autoFocus` should work since version 6.3.0 (merged [PR](https://github.com/okta/okta-signin-widget/pull/2450))

**Comment by kito99** [comment]
What if you use ClassicEngine?
